examples/autonat/src/bin/autonat_server.rs

```diff
@@ -37,7 +39,11 @@ struct Opt {
 
 #[async_std::main]
 async fn main() -> Result<(), Box<dyn Error>> {
-    env_logger::init();
+    let env_filter = EnvFilter::builder()
+        .with_default_directive(LevelFilter::DEBUG.into())
```

Comment:
I think the default is `INFO`, right? Is logging on debug worth the extra boilerplate here or should we just use `EnvFilter::from_default_env()` and save a few lines in all examples?

Reply:
We could replace this with the following which is quite a bit shorter:

```rust
tracing_subscriber::fmt().with_env_filter(EnvFilter::from_default_env()).init();
```